### ISSUE TITLE
Website updates after ethCC

### ISSUE THREAD

**Issue by ukorvl** [NilFoundation/nil.foundation#175]
https://miro.com/app/board/uXjVKweu-Dg=/ miro board with all details.

- [x] Dev stand for the site. We need dev stand in the infrastructure to present our updates to marketing and only after that we can proceed with deployment to production. Actually we have role for website, need setup ansible from dev branch.
- [x] Navigation. Remove product pages links and Research link
- [x] Home page. Update texts accordingly with the miro, CTA link, image
- [x] "Join our community" section. This section is repeated across the pages. It's needed to change documents icon. New icon 
![image](https://github.com/user-attachments/assets/5527ab57-cb31-440f-96a9-8039e6d3e643)
- [x] Blog page. Remove tags, update categories and texts
- [x] Remove "Research" page. The links to the documents will be added in a new blog post.
- [x] "About" page update. Update texts and sections
- [x] Careers. Update texts and merge `/careers` and `/careers/jobs`. The jobs list from `/careers/jobs` should go right after the heading and the text @ukorvl 
- [x] Documentation - remove docs of old products and left only nil documentation. (?)

**Comment by ukorvl** [comment]
Merging of careers and jobs pages better be done in a separate pull request.

**Comment by ukorvl** [comment]
Done
